### ISSUE TITLE
Installation fails on 5.6 when sql_mode includes ONLY_FULL_GROUP_BY

### ISSUE THREAD

**Issue by solomonty** [MarkLeith/mysql-sys#56]
ERROR 1055 (42000) at line 47 in file: './views/p_s/processlist.sql': 'performance_schema.pps.PROCESSLIST_ID' isn't in GROUP BY

````
@@sql_mode
ONLY_FULL_GROUP_BY,NO_AUTO_VALUE_ON_ZERO,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
````

    # Change history for the MySQL sys schema
    ## 1.4.0 (09/03/2015)

**Comment by solomonty** [comment]
The MySQL server is running version 5.6.23.

**Comment by MarkLeith** [comment]
Fix pushed as part of the 1.5.0 release.
